mx.py

```diff
@@ -581,6 +581,7 @@ def __init__(self, parents=None):
                                 "projects and store it in the given <file>. If <file> is 'default', the compilation database will "
                                 "be stored in the parent directory of the repository containing the primary suite. This option "
                                 "can also be configured using the MX_COMPDB environment variable. Use --compdb none to disable.")
+        self.add_argument('--arch', action='store', dest='arch', help='force use the specified architecture', choices=['arm64', 'aarch64', 'amd64', 'sparcv9'])
```

Comment:
Why allow `arm64` in there? we usually canonicalize to `aarch64`

Reply:
Agreed, it's just because Apple calls their processors "arm64" (IIRC). We could remove it if we don't like to have an alias (as far as I know they are the same).

Reply:
Let's keep mx arches canonical until there's a good reason not to.